schema/record-syntax.shacl

```diff
@@ -29,11 +48,27 @@ rec:RecordShape
         sh:message "A record can be the subrecord of at most one record";
         sh:severity sh:Violation;
     ] ,
+    [
+        sh:path rec:replaces;
+        sh:class rec:Record;
+        sh:minCount 0;
+        sh:maxCount 1;
```

Comment:
Is this `maxCount` correct? The `sh:message` seem to indicate you could replace any number of records.

Reply:
Good catch